db/db_impl.cc

```diff
@@ -5096,8 +5052,6 @@ void DBImpl::MaybeFlushColumnFamilies() {
     alive_log_files_.begin()->getting_flushed = true;
   }
 
-  WriteContext context;
```

Comment:
WriteContext is passing from WriteImpl() instead and destruct without holding mutex lock.